csaf/remotevalidation.go

```diff
@@ -55,9 +56,37 @@ type outDocument struct {
 	Document any    `json:"document"`
 }
 
+// rtest is the result of the remote tests
+// recieved by the remote validation service.
+type rtest struct {
+	Valid   bool        `json:"isValid"`
+	Warning []rwarnings `json:"warnings"`
+	Error   []rerrors   `json:"errors"`
+	Info    []rinfos    `json:"infos"`
+}
```

Comment:
The test name should be returned as well `name`. Otherwise, it is hard to figure out which test is which.
```suggestion
        Name    string    `json:"name"`
}

Reply:
Please also sort keys (errors, warning, infos) it to match https://github.com/csaf-poc/csaf_distribution/issues/324#issuecomment-1421665324

Reply:
Has now been done

Reply:
Thanks for pointing that out - I didn't check the order in the comment. The order should match: https://github.com/csaf-poc/csaf_distribution/issues/348#issue-1607401382